lib/stream.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
-module.exports = Stream;
-
 const EE = require('events');
 const util = require('util');
 
+function Stream() {
+  EE.call(this);
+}
 util.inherits(Stream, EE);
```

Comment:
Or why not just move the file as-is to internal/* and have this just `require()` and return that?